helm-charts/flink-historyserver/templates/efs.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: efs-flink-history
+parameters:
+  provisioningMode: efs-ap
+  fileSystemId: "{{- .Values.aws_efs_file_system_id }}"
```

Comment:
Generally helm uses camelCase, so let's call this something like `.Values.efs.fileSystemId`?

Reply:
yep, solved